text-ui-test/input.txt

```diff
@@ -0,0 +1,3 @@
+todo something
+deadline eat /by tmr
+event picnic /at 9 Aug
```

Comment:
I think it will be great to add testing for other commands such as list as well 😄